Gemfile.lock

```diff
@@ -0,0 +1,15 @@
+GEM
```

Comment:
do we need this file ?

Reply:
No, Gemfile.lock isn't required. Forgot to remove that.

Reply:
@YashKumarVerma Have removed that in the pr now.